### PR TITLE
Add scheduled CI workflow for opening RC's

### PR DESCRIPTION
**Pull request by dannyphillips**
### Changelog
- Switched to using workflows for CI
- Created release_candidate script for opening RC branch/PR
- Added nightly workflow for opening RC from develop

### Next Steps
- Breakout `build_test_deploy` workflow into many jobs
- Create

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -55,11 +67,28 @@ jobs:
...
               fi
             fi
+
+workflows:
+  version: 2
+    
+  release_candidate_workflow:
+    triggers:
+      - schedule:
+          cron: "0 8 * * 1" ## Weekly at 8:00am on Monday mornings
```
Comment (mboperator) [inline .circleci/config.yml]: Wow this is sweet.

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+  ## Open an release candidate branch/PR from develop on a weekly schedule.
     docker:
-      - image: circleci/node:latest
+      - image: circleci/node:8.8.1
+
+    steps:
+      - checkout
+      - run:
+          name: Run RC Script
+          command: echo "ran the RC script"
```
Comment (mboperator) [inline .circleci/config.yml]: Just to clarify: this branch sets up the scheduled calling of the RC script, but the script itself isn't implemented yet right?

**Comment by dannyphillips** [comment]
Actually, I'm gonna change this branch to still put us in workflows, but not the release candidate stuff

**Comment by dannyphillips** [comment]
Closing for now, we'll revisit after we implement workflows and have solved the semver management for RC's.

**Pull request closed without merging**